src/transport.ts

```diff
@@ -184,3 +184,38 @@ export class SequenceNumberMixer<V> {
     return value;
   }
 }
+
+export class InterceptingTransport implements ConnectionTransport {
```

Comment:
Where is this one used?

Reply:
Nowhere just yet, but looks generally useful?